### PR TITLE
Restructure chain-evolution technology

### PR DESCRIPTION
**Pull request by Dfinity-Bjoern**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

File: how-it-works/3_chainevolution/01_scalability.subpage.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+Creating a new subnet has two steps.  1) Adding new nodes to the Internet Computer,  and  2) Creating a subnet with the available nodes
+Anyone can purchase the node hardware and add it to the Internet Computer by following the [node provider onboarding process](https://wiki.internetcomputer.org/wiki/Node_Provider_Onboarding). 
+
+We now describe how to create a new subnet with the available nodes. The Internet Computer has a decentralized governance system called Network Nervous System (NNS). Essentially, NNS consists of a group of canisters that manage the Internet Computer. In the NNS, there is a component called “registry”, which stores all the configuration of the Internet Computer. The registry has a record for each subnet which includes a protocol version, list of nodes in the subnet, cryptographic key material to be used by the subnet, a Catch Up Package (CUP), etc.
```
Comment (sesi200) [inline how-it-works/3_chainevolution/01_scalability.subpage.md:19]:
```suggestion
We now describe how to create a new subnet with the available nodes. The Internet Computer has a decentralized governance system called Network Nervous System (NNS). Essentially, NNS consists of a group of canisters that manage the Internet Computer. In the NNS, there is a component called “registry”, which stores all the configurations of the Internet Computer. The registry has a record for each subnet which includes a protocol version, list of nodes in the subnet, cryptographic key material to be used by the subnet, a Catch Up Package (CUP), etc. 
```

Comment (sesi200) [inline how-it-works/3_chainevolution/01_scalability.subpage.md:19]: >he registry has a record for each subnet which includes [...] cryptographic key material to be used by the subnet

Can this be misconstrued as the NNS holding private keys of another subnet?

File: how-it-works/3_chainevolution/01_scalability.subpage.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+We now describe how to create a new subnet with the available nodes. The Internet Computer has a decentralized governance system called Network Nervous System (NNS). Essentially, NNS consists of a group of canisters that manage the Internet Computer. In the NNS, there is a component called “registry”, which stores all the configuration of the Internet Computer. The registry has a record for each subnet which includes a protocol version, list of nodes in the subnet, cryptographic key material to be used by the subnet, a Catch Up Package (CUP), etc. 
+
+To add a new subnet, one has to submit a “proposal” in the NNS to add record for a new subnet in the registry. The proposal consists of the list of nodes to be included in the new subnet. The proposal can be voted by anyone who staked their ICP tokens. If a majority of voters accept the proposal, then the registry canister computes the cryptographic key material to be used by the new subnet and a catch up package containing the genesis block. The registry canister then adds a record containing the configuration of the subnet.
```
Comment (sesi200) [inline how-it-works/3_chainevolution/01_scalability.subpage.md:21]:
```suggestion
To add a new subnet, one has to submit a “proposal” in the NNS to add a record for a new subnet in the registry. The proposal consists of the list of nodes to be included in the new subnet. The proposal can be voted by anyone who staked their ICP tokens. If a majority of voters accept the proposal, then the registry canister computes the cryptographic key material to be used by the new subnet and a catch up package containing the genesis block. The registry canister then adds a record containing the configuration of the subnet. 
```

Comment (sesi200) [inline how-it-works/3_chainevolution/01_scalability.subpage.md:21]: > the registry canister computes the cryptographic key material to be used by the new subnet

Same issue here

File: how-it-works/3_chainevolution/03_upgrades.subpage.md [hunk 3]
```diff
@@ -7,7 +7,30 @@ slug: upgrades
...
+
+While upgrading a blockchain protocol is extremely crucial for its success, most blockchains including Bitcoin and Ethereum are not designed to do so. This is primarily because blockchains are not controlled by a single authority. Every upgrade proposal has to be evaluated by the community. However, the community is usually split on the proposals. There is no quick and formal framework to finalize the decisions and build new features. Upgrades to the protocol potentially cause a fork in the network. As a result, upgrading a blockchain protocol could take years of joint effort by the community. Ethereum went through only [18 protocol upgrades in a 7.5 year time span](https://ethereum.org/en/history/).
+
+The Internet Computer is a unique blockchain that is designed to be easily upgradeable with a minimal user-perceived downtime and without any forks while still requiring consensus by the community for each upgrade. Within 1.5 years after genesis, the Internet Computer has upgraded many times adding crucial features such as Deterministic Time Slicing, Bitcoin Integration, Service Nervous System, HTTPS Outcalls, Threshold ECDSA, Increased Stable Memory, etc.
```
Comment (sesi200) [inline how-it-works/3_chainevolution/03_upgrades.subpage.md:14]:
```suggestion
The Internet Computer is a unique blockchain that is designed to be easily upgradeable with a minimal user-perceived downtime and without any forks while still requiring consensus by the community for each upgrade. Within 1.5 years after genesis, the Internet Computer has upgraded many times, adding crucial features such as Deterministic Time Slicing, Bitcoin Integration, Service Nervous System, HTTPS Outcalls, Threshold ECDSA, Increased Stable Memory, etc. 
```

File: how-it-works/3_chainevolution/03_upgrades.subpage.md [hunk 4]
```diff
@@ -7,7 +7,30 @@ slug: upgrades
...
+
+We now describe how a change in registry version upgrades the Internet Computer. Protocol upgrades are done on a per-subnet basis. Each subnet is run by many nodes. Each node runs 2 processes — (1) the Replica and (2) the Orchestrator. The Replica consists of the 4-layer software stack that maintains the blockchain. The Orchestrator downloads and manages the replica software. The orchestrator regularly queries the NNS registry for any updates. If there is a new registry version, the orchestrator downloads the corresponding replica software and informs about it to the replica. 
+
+All the nodes in the subnet together run a consensus protocol. In each round, one of the nodes (called the block maker) proposes a block and the other nodes notarize the block (approve the block by signing it). In every block, the blockmaker includes the latest registry version it downloaded from the registry. Other nodes notarize the block only when they have the referenced registry available.
```
Comment (sesi200) [inline how-it-works/3_chainevolution/03_upgrades.subpage.md:24]:
```suggestion
All the nodes in the subnet together run a consensus protocol. In each round, one of the nodes (called the block maker) proposes a block and the other nodes notarize the block (approve the block by signing it). In every block, the block maker includes the latest registry version it downloaded from the registry. Other nodes notarize the block only when they have the referenced registry available. 
```

**Review by sesi200 — CHANGES_REQUESTED** [review]

File: how-it-works/3_chainevolution/01_scalability.subpage.md (hunk 1 above)
Comment (Dfinity-Bjoern) [reply how-it-works/3_chainevolution/01_scalability.subpage.md]: That may need some clarification indeed. I'll do that.

File: how-it-works/3_chainevolution/01_scalability.subpage.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+We now describe how to create a new subnet with the available nodes. The Internet Computer has a decentralized governance system called Network Nervous System (NNS). Essentially, NNS consists of a group of canisters that manage the Internet Computer. In the NNS, there is a component called “registry”, which stores all the configurations of the Internet Computer. The registry has a record for each subnet which includes a protocol version, the list of nodes in the subnet, protocol configuration parameters, etc. 
+
+To add a new subnet, one has to submit a “proposal” in the NNS to add a record for a new subnet in the registry. The proposal consists of the list of nodes to be included in the new subnet. The proposal can be voted by anyone who staked their ICP tokens. If a majority of voters accept the proposal, then the registry canister instructs the NNS subnet to generate - in a fully decentralized way using [chain-key cryptography]() - computes the cryptographic key material to be used by the new subnet and a catch up package containing the genesis block. The registry canister then adds a record containing the configuration of the subnet.
```
Comment (sesi200) [inline how-it-works/3_chainevolution/01_scalability.subpage.md:21]:
```suggestion
To add a new subnet, one has to submit a “proposal” in the NNS to add a record for a new subnet in the registry. The proposal consists of the list of nodes to be included in the new subnet. The proposal can be voted by anyone who staked their ICP tokens. If a majority of voters accept the proposal, then the registry canister instructs the NNS subnet to generate - in a fully decentralized way using [chain-key cryptography](/how-it-works/chain-key-technology/) - computes the cryptographic key material to be used by the new subnet and a catch up package containing the genesis block. The registry canister then adds a record containing the configuration of the subnet. 
```

**Review by sesi200 — APPROVED** [review]

**Pull request merged**